scripts/migration/renamings.js

```diff
@@ -1011,6 +1011,39 @@ const renamings = {
       },
     },
   },
+  '7.20211209.0': {
+    'Blockly': {
+      exports: {
+        DRAG_RADIUS: {
+          module: 'Blockly.config',
+          export: 'config.dragRadius',
```

Comment:
I had originally envisioned that `export` would only ever be a plain identifier, but obviously for things that have been moved on to an options object that won't be enough.  I think the way you have done it here is a perfectly reasonable approach that will take approximately zero extra work for the script to handle.